apps/scheduler/internal/timers/online.go

```diff
@@ -2,31 +2,64 @@ package timers
 
 import (
 	"context"
-	"sync"
 	"time"
 
-	"gorm.io/gorm"
-
 	"github.com/google/uuid"
 	"github.com/guregu/null"
 	"github.com/nicklaw5/helix/v2"
 	"github.com/samber/lo"
 	loParallel "github.com/samber/lo/parallel"
-	"github.com/satont/twir/apps/scheduler/internal/types"
+	"go.uber.org/zap"
+	"gorm.io/gorm"
+
 	model "github.com/satont/twir/libs/gomodels"
 	"github.com/satont/twir/libs/twitch"
-	"go.uber.org/zap"
+
+	"github.com/satont/twir/apps/scheduler/internal/types"
 )
 
+func scanChatters(
+	ctx context.Context,
+	broadcasterId string,
+	services *types.Services,
+) ([]helix.ChatChatter, error) {
+	twitchClient, err := twitch.NewUserClientWithContext(ctx, broadcasterId, *services.Config, services.Grpc.Tokens)
+	if err != nil {
+		zap.S().Error(err)
```

Comment:
Error will be logged twice becase on the parent function you also doing logging.

Reply:
removed